lesson03/main.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.getElementById("js-target");
+
+const li = [];
+const a = [];
+const img = [];
```

Comment:
IMO
I don't think you need to create an empty array here.
 I think it's better to make an array of elements to be put in the src attribute of img first, and then loop through it.
```suggestion
const imgSrcs = ["/img/bookmark.png", "/img/messege.png" ];
```

Reply:
thank you mai-san :)
thank you teachere :)
I will define objects or something at first.

Reply:
I fixed my code :)
afe9a45f878523ce5b6fd09769088db7fcd2c0a6